Makefile

```diff
@@ -96,7 +97,13 @@ lint: $(GOLANGCI_LINT)
 
 fmt:
 	@echo gofmt		# Show progress, real gofmt line is too long
-	find pkg cmd test -name '*.go' | xargs -r gofmt -s -l -w
+	@if [ $(OS_NAME) == 'Linux' ] ; then \
+		find pkg cmd test -name '*.go' | xargs -r gofmt -s -l -w; \
+	fi
+	# BSD (MacOS) xargs does not have option -r (--no-run-if-empty)
+	@if [ $(OS_NAME) = 'Darwin' ] ; then \
+		find pkg cmd test -name '*.go' | xargs gofmt -s -l -w; \
+	fi
```

Comment:
The checks should use now lowercase, too, e.g. `linux`.

Reply:
done